docs/pages/pmd/userdocs/tools/maven.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Maven PMD Plugin
+permalink: /pmd_userdocs_tools_maven.html
+last_updated: August 2017
+author: >
+    Miguel Griffa <mikkey@users.sourceforge.net>,
+    Romain PELISSE <belaran@gmail.com>,
+    Andreas Dangel <andreas.dangel@adangel.org>
+---
+
+## Maven 2 and 3
+
+### Running the pmd plugin
+
+#### report
```

Comment:
Well, it was before :) But I think, you're right, it doesn't make sense. Maybe it was thought of something like the maven goal name, but this is actually either "pmd" or "check"...